include/entity.h

```diff
@@ -94,9 +94,10 @@ typedef struct Entity_ {
     /*0x24*/ s16 speed;
     /*0x26*/ u8 spriteAnimation[3];
     /*0x29*/ struct {
-    /*    */     u8 b0 : 3; // 1-4
-    /*    */     u8 b1 : 3; //   8
-    /*    */     u8 b2 : 2; //0x40
+    /*    */     u8 b0 : 3; // 0-2
```

Comment:
Why the change here? I find the hex values more useful since the compiler usually will form an immediate value (so this is what decompilers display) and not bit shifts.